apps/api-gateway/src/credential-definition/credential-definition.controller.ts

```diff
@@ -61,12 +61,13 @@ export class CredentialDefinitionController {
   async getCredentialDefinitionBySchemaId(
```

Comment:
Summary and description should not be same

Reply:
Resolved